Python/operators.py

```diff
@@ -1,54 +1,49 @@
 import warnings
 
-import xarray as xr
 import numpy as np
 
+
 def check_keys(data_cube):
     rlat = 'rlat' in data_cube.keys()
     rlon = 'rlon' in data_cube.keys()
     run = 'run' in data_cube.keys()
     dv = 'dv' in data_cube.keys()
     if not rlat and rlon and run and dv:
-        raise KeyError("CanRCM4 ensemble is missing 'run', 'rlon', 'rlat', or 'dv'. Please ensure the model is properly \
-                        gridded with these names.")
+        raise KeyError(("CanRCM4 ensemble is missing 'run', 'rlon',\
```

Comment:
Would probably be helpful to tell the user *which* it's missing. 

```python
required_keys = {'rlat', 'rlon', 'run', 'dv'}
actual_keys = set(data_cube.keys())
if required_keys <= actual_keys:
    raise KeyError("CanRCM4 ensembles is missing keys {}".format(required_keys - actual_keys))
```

Reply:
Nice. I'm going to add this to `datacube.py` instead. I think it makes more sense there. I will only use this on function where I specifically use keys.